src/assets/styles/other/_tn-styles.scss

```diff
@@ -761,61 +763,63 @@ $primary-dark: darken(map-get($md-primary, 500), 8%);
     color: var(--fg2);
   }
 
-  .mat-button {
+  .mat-mdc-button,
+  .mdc-button {
     @include variable(font-family, --font-family-body, $font-family-body, !important);
     background: var(--btn-default-bg);
     font-weight: 700 !important;
     text-transform: capitalize !important;
   }
 
-  .mat-stroked-button {
+  .mdc-button--outlined {
     background-color: transparent !important;
     border: 1px solid currentColor !important;
-    border-radius: 25px;
-    line-height: 28px;
+    border-radius: 25px !important;
+    height: 30px;
+    line-height: 30px;
 
     &.fn-theme-primary {
-      @include variable(color, --primary, $primary);
+      @include variable(color, --primary, $primary, !important);
```

Comment:
Why all the `!important`s? Not sure how to test if they are needed?

Reply:
to override default styles